docs/js/react.js

```diff
@@ -711,6 +711,7 @@ var isUnitlessNumber = {
 
   // SVG-related properties
   fillOpacity: true,
+  stopOpacity: true,
```

Comment:
Please revert this, this file is the release version of 0.13.3 and should not get modified directly.